includes/utilities.php

```diff
@@ -150,6 +155,8 @@ function insert_redirect( $from, $to, $status_code, $post_id = 0 ) {
 	 */
 	$to = apply_filters( 'hm_redirects_pre_save_to_url', $to, $from, $status_code, $post_id );
 
+	$preserve_parameters = filter_var( $preserve_parameters, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE );
```

Comment:
Not sure how necessary `filter_var()` is here, also the `$preserve_parameters` value should be passed to the filters above to keep it consistent.